BaseFilter/sections/general_url.txt

```diff
@@ -15,6 +15,8 @@
 ! Banner names
 !
 !
+! Lucky visitor and video scam ex. https://1821.antbenjaw.live/jiehxxgr/?u=1nup806&o=0wywy2l&t=...
+/^https?:\/\/(?!www)[0-9a-z]*\.?[-0-9a-z]{4,}\.[a-z]{2,11}\/(?:[0-9a-z]{6,8}\/)?\/?\?[ou]=[0-9a-z]{7}&[ou]=[0-9a-z]{7}/$document,match-case,domain=beauty|buzz|com|de|fun|guru|im|info|life|live|mobi|motorcycles|online|pw|quest|shop|site|skin|space|top|us|xyz|yachts
```

Comment:
`com|de` makes whole `$domain` redundant.

Reply:
Okay, will remove the domain part.

Reply:
Should not `&t=` be matched?